spec/fixtures/approvals/rssreader/fetch_user/gets_articles_for_user.approved.txt

```diff
@@ -1 +1 @@
-1
\ No newline at end of file
+10
```

Comment:
these change because I re-fetched all live RSS feeds, so the articles have increased by now :D